packages/nimble-components/src/dialog/specs/README.md

```diff
@@ -157,7 +198,7 @@ By using the native `dialog` element, we get good a11y behavior without having t
 -   ESC key closes the dialog
 -   The native dialog defaults to the a11y role `dialog` (but we will override this, as discussed below)
 
```

Comment:
How will the subtitle interact from aria? I could kinda imagine it being part of the title and should be read out as part of the aria-lable.
I could also maybe see it as part of the aria-describedby.
Maybe it's something to nail down in implementation but we should make sure it's used appropriately somehow.